app/src/main/java/org/covidwatch/android/data/ContactEvent.kt

```diff
@@ -10,7 +10,7 @@ import java.util.*
 @TypeConverters(DateConverter::class, UploadStateConverter::class)
 class ContactEvent {
     @PrimaryKey
-    var identifier: String = UUID.randomUUID().toString().toUpperCase()
+    var identifier: String = UUID.randomUUID().toString().toUpperCase(Locale.ROOT)
```

Comment:
What does this do?

Reply:
It makes sure the string conversion yields the same result for all cultures. Since string conversion (as well as e.g. string comparison) is culture sensitive, I added Locale.ROOT, which is a sort of "culture independent dummy culture". This was a hint by Android Studio,